mlrun/feature_store/feature_set.py

```diff
@@ -356,6 +373,7 @@ def add_aggregation(
         operations,
         window,
         period=None,
+        step_name=None,
```

Comment:
That is a backwards incompatible change, since the 7th parameter used to be `after` and now it's `state_name`, use here the same thing you did in the other places

Reply:
Yes, you're correct. I missed that one.